distributions/aws/infra_configs/scripts/config.yaml

```diff
@@ -0,0 +1,10 @@
+cognitoUserpool:
+  name: kubeflow-userpool
+kubeflow:
+  region: us-east-1
+route53:
+  rootDomain:
+    hostedZoneId: XXXXX123ZZZZ88XX33WW
```

Comment:
what does this mean ?

Reply:
these are just dummy values. The user needs to replace all of them. should I make them all XXX?

Reply:
yeah i think we should leave these fields empty so it clear which fields to change

including an example config in the README would be helpful for showing how to configure these values

Reply:
Sg, will updated this PR